src/integrationTest/java/uk/gov/hmcts/reform/pip/rules/controllers/ControllerTest.java

```diff
@@ -51,13 +51,43 @@ public void publicationThatExistsTest() throws Exception {
         final int publicationID = 1;
         assertEquals(publicationID, publication.getPublicationId(), "Publication ID is " + publicationID);
         assertEquals(1, publication.getCourtHearingsList().size(), "The array contains a single publication");
+    }
+
+    @DisplayName("Should return a 200, with a publication that contains a single court list")
+    @Test
+    public void publicationThatExistsHearingsListTest() throws Exception {
+        MvcResult response = mockMvc.perform(get("/publication/1")).andExpect(status().isOk()).andReturn();
+
+        assertEquals(200, response.getResponse().getStatus(),
+                     "Check that a 200 status is returned is returned");
+
+        ObjectMapper objectMapper = new ObjectMapper();
+        objectMapper.registerModule(new JavaTimeModule());
+        Publication publication = objectMapper.readValue(
+            response.getResponse().getContentAsByteArray(), Publication.class);
 
         List<CourtHearings> courtHearingsList = publication.getCourtHearingsList();
         assertThat(courtHearingsList.get(0).getCourtId()).as("Get first court id").isEqualTo(1);
+    }
+
+    @DisplayName("Should return a 200, with a publication that contains a multiple hearings")
+    @Test
+    public void publicationThatExistsHearingsTest() throws Exception {
+        MvcResult response = mockMvc.perform(get("/publication/1")).andExpect(status().isOk()).andReturn();
+
+        assertEquals(200, response.getResponse().getStatus(),
+                     "Check that a 200 status is returned is returned");
+
+        ObjectMapper objectMapper = new ObjectMapper();
+        objectMapper.registerModule(new JavaTimeModule());
+        Publication publication = objectMapper.readValue(
+            response.getResponse().getContentAsByteArray(), Publication.class);
+
+        List<CourtHearings> courtHearingsList = publication.getCourtHearingsList();
+        List<Hearing> hearings = courtHearingsList.get(0).getHearingList();
 
-        List<Hearing> courtHearings = courtHearingsList.get(0).getHearingList();
-        assertThat(courtHearings.get(0).getHearingId()).as("Get first hearing id").isEqualTo(1);
-        assertThat(courtHearings.get(1).getHearingId()).as("Get second hearing id").isEqualTo(2);
+        assertThat(hearings.get(0).getHearingId()).as("Get first hearing id").isEqualTo(1);
+        assertThat(hearings.get(1).getHearingId()).as("Get second hearing id").isEqualTo(2);
     }
```

Comment:
what happens if you do `/publication/2`?

Reply:
or is that tested elsewhere?

Reply:
/2 should return a 404.

Indeed. That's being covered on in the test on line 96

Reply:
Yeah as Kurt states a not found test would be good

Reply:
ahh yeah it was not showing up when expanded